### PR TITLE
Speed mp values

### PR DESCRIPTION
**Pull request by Dak2896**
Nucleus: 100 MP
Cytoplasm: 35 MP
Chloroplast: 55 MP
Oxy: 85 MP
Mitochondrion: 55 MP
Vacuole: 50 MP
Flagellum: 55 MP
Chemoplast: 45
Nitrogen: 50 MP
Respiratory protein: 45 MP
Photosynthetic Protein: 55 MP
ChemoSynthisizing Protein: 45 MP
Protoplasm: 55 MP
NitrogenFixation Protein: 45 MP

### CODE REVIEW

**Comment by Dak2896** [comment]
I didn't disable cheat? is for that the error i think

File: scripts/microbe_stage/organelle_table.as [hunk 1]
```diff
@@ -464,15 +464,15 @@ void setupOrganelles(){
...
+    chemoplast.mpCost = 45;
     chemoplast.initialComposition = {
         {"phosphates", 5},
         {"ammonia", 5}
     };
     chemoplast.components = {
         processorOrganelleFactory(1.0f),
     // Chemoplast takes 2 hexes, so allowed storage of 2 cytooplasm
-    storageOrganelleFactory(10.0f)
+    storageOrganelleFactory(0.0f)
```
Comment (Untrustedlife) [inline scripts/microbe_stage/organelle_table.as]: if it stores 0 just remove that factory from the list,

Comment (Dak2896) [reply scripts/microbe_stage/organelle_table.as]: my bad i didn't restore that value

**Comment by hhyyrylainen** [comment]
The jenkins build is not working because the fix for that is on the branch that untrustedlife is working on and where I dumped some random stuff. It's not serious.

**Pull request merged**